### PR TITLE
UI tweaks

### PR DESCRIPTION
**Pull request by tarheel**
image

### CODE REVIEW

File: src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml [hunk 1]
```diff
@@ -539,7 +539,7 @@
             <VBox>
               <VBox maxWidth="460.0" styleClass="bordered-box">
                 <HBox spacing="4.0">
-                  <Label prefWidth="212.0" text="Overvote Rule *"/>
+                  <Label prefWidth="100.0" text="Overvote Rule *"/>
```
Comment (HEdingfield) [inline src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml:542]: I'll need to check this in Linux tomorrow before approving.

Comment (HEdingfield) [reply src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml:542]: Yeah, this cuts things off in Linux. Please revert to what it was before.

Comment (HEdingfield) [reply src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml:542]: 120.0 also works ok, which is closer, but isn't in line with the other input box. Either that or 212 is ok with me.

**Review by HEdingfield — APPROVED** [review]

File: src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml (hunk 1 above)
Comment (tarheel) [reply src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml]: OK, going with 120.

**Pull request merged**